test/unit/test_cli.py

```diff
@@ -73,11 +74,19 @@ def test_execute_command_max_retry(self, mock_open, mock_network_error):
     def test_set_audit_api_url(self, mock_getuser_func):
         """Test sending data to given url"""
         mock_getuser_func.return_value = 'mockuser'
-        expected_body = '{"directory": "test/path", "status": "FAILED", "run_by": "mockuser", "output": []}'
+
+        expected_body = '{"directory": "/test/helpers/mock_directory/config/.tf_wrapper", ' \
```

Comment:
Use `json.dumps` to create the json instead of hardcoding the json string

Reply:
Fixed